lib/NewFangle.pm

```diff
@@ -14,10 +14,32 @@ package NewFangle {
 =head1 SYNOPSIS
 
  use NewFangle;
- my $app = NewFangle::App->new('MyApp', $license_key);
- my $txn = $app->start_transaction('my transaction');
+ my $app = NewFangle::App->new({app_name => 'MyApp', license_key => $license_key});
+ my $txn = $app->start_web_transaction('my transaction');
  $txn->end;
 
+Or using a L<NewFangle::Config>:
+
+ use NewFangle;
+ my $config = NewFangle::Config->new(
+   app_name => 'MyApp',
+   license_key => $license_key,
+ );
+ my $app = NewFangle::App->new($config);
+ my $txn = $app->start_web_transaction('my transaction');
+
+On your dashboard side, you will get:
```

Comment:
Can you put this in the `html`, otherwise the textual renderings will just have this statement without the image.